pages/docs/concepts/server.md

```diff
@@ -0,0 +1,6 @@
+
+## What is a Server?
+APIs use servers to receive requests from a user and responds back with the required information. In a nutshell, a server is a middle man between a user and the stored information inside a database. The communication between a user and the database happens in the form of HTTP messages. One thing to note, servers cannot make any requests, and the user can’t respond as all interactions are instigated by the user.
+
+## Purpose of Servers in Event-Driven Architectures
```

Comment:
I'd dedicate this whole section to the concept of message broker, which is a special kind of server but it's the most common in EDA.